src/ununpack/ui/ui-reunpack.php

```diff
@@ -167,21 +167,19 @@ function AgentAdd ($uploadpk, $Depends=NULL, $priority=0)
     $row = pg_fetch_assoc($result);
     pg_free_result($result);
 
-    if (!empty($row)){
-      $jobpk = $row['job_pk'];
-    } else {
-      $result = pg_query($PG_CONN, $SQLInsert);
-      DBCheckResult($result, $SQLInsert, __FILE__, __LINE__);
-      $row = pg_fetch_assoc($result);
-      pg_free_result($result);
-      $SQLcheck = "SELECT job_pk FROM job WHERE job_upload_fk = '$uploadpk'"
-        . " AND job_name = '$Job_name' AND job_user_fk = '$user_fk';";
-      $result = pg_query($PG_CONN, $SQLcheck);
-      DBCheckResult($result, $SQLcheck, __FILE__, __LINE__);
-      $row = pg_fetch_assoc($result);
-      pg_free_result($result);
-      $jobpk = $row['job_pk'];
+    if (empty($row)) {
+        $result = pg_query($PG_CONN, $SQLInsert);
+        DBCheckResult($result, $SQLInsert, __FILE__, __LINE__);
+        $row = pg_fetch_assoc($result);
+        pg_free_result($result);
+        $SQLcheck = "SELECT job_pk FROM job WHERE job_upload_fk = '$uploadpk'"
+            . " AND job_name = '$Job_name' AND job_user_fk = '$user_fk';";
+        $result = pg_query($PG_CONN, $SQLcheck);
+        DBCheckResult($result, $SQLcheck, __FILE__, __LINE__);
+        $row = pg_fetch_assoc($result);
+        pg_free_result($result);
```

Comment:
Please fix the indentation
```suggestion
    if (empty($row)) {
      $result = pg_query($PG_CONN, $SQLInsert);
      DBCheckResult($result, $SQLInsert, __FILE__, __LINE__);
      $row = pg_fetch_assoc($result);
      pg_free_result($result);
      $SQLcheck = "SELECT job_pk FROM job WHERE job_upload_fk = '$uploadpk'"
        . " AND job_name = '$Job_name' AND job_user_fk = '$user_fk';";
      $result = pg_query($PG_CONN, $SQLcheck);
      DBCheckResult($result, $SQLcheck, __FILE__, __LINE__);
      $row = pg_fetch_assoc($result);
      pg_free_result($result);
```